apps/andi/config/integration.exs

```diff
@@ -47,12 +47,13 @@ config :andi, AndiWeb.Endpoint,
   check_origin: false
 
 # for auth0 login use, add to Endpoint config above
-# https: [
-#   port: 4443,
-#   otp_app: :andi,
-#   keyfile: "priv/key.pem",
-#   certfile: "priv/cert.pem"
-# ]
+# config :andi, AndiWeb.Endpoint,
```

Comment:
put the full config in here to make it easier to just toggle on and off. We could probably just use a mix arg for this too

Reply:
I actually ended up just turning this on permanently